src/catalog/protos/hashes.json

```diff
@@ -21,6 +21,6 @@
   },
   {
     "name": "objects_v64.proto",
-    "md5": "e914c3020f9625f33d2ac89579020faf"
+    "md5": "35ed48a2d63a9c17d14b3ed2c648abca"
```

Comment:
_Technically_ your change to the protobuf files is forwards and backwards compatible, but the way we've structured the catalog migration is the hashes for the `object_X.proto` files should never change. The goal is they exist as a snapshot in time of what was serialized to the catalog.

If you don't mind refactoring this a bit, can you create a new `objects_v65.proto` that has your changes, and leave `objects_v64.proto` untouched? Then add `v65` to the [`objects!`](https://github.com/MaterializeInc/materialize/blob/main/src/catalog/src/durable/upgrade.rs#L184-L191) macro, and bump the `CATALOG_VERSION: u64 = 65`. Finally, add an upgrade [here](https://github.com/MaterializeInc/materialize/blob/main/src/catalog/src/durable/upgrade.rs#L288) for v64, and it'll just be a no-op upgrade like [v63_to_v64](https://github.com/MaterializeInc/materialize/blob/main/src/catalog/src/durable/upgrade/v63_to_v64.rs).

I know this is a lot of boilerplate for a change that should be quite easy, but by sticking to this framework it's made it easier to reason about how the catalog evolves. More than happy to do some pair programming here if it helps!

Reply:
Yeah for sure!